src/components/Deliberation/SecondRound.js

```diff
@@ -0,0 +1,170 @@
+import React, { Component } from "react";
+
+import Button from "react-bootstrap/Button";
+import Table from "react-bootstrap/Table";
+import Modal from "react-bootstrap/Modal";
+import Form from "react-bootstrap/Form";
+
+import { withAuthorization } from "upe-react-components";
+
+import { Container } from "../../styles/global";
+import { isAdmin } from "../../util/conditions";
+
+class SecondRound extends Component {
+  state = {
+    showModal: false,
+    currentApplicationId: null,
+    currentMeetingStatus: false,
+    currentContributionStatus: false,
+    currentName: "",
+  };
+
+  render() {
+    const { applications, settings } = this.props;
+
+    const provisionalMemberApplications = applications.filter((app) =>
+      app.hasOwnProperty("provisional")
+    );
+
+    const {
+      showModal,
+      currentApplicationId,
+      currentContributionStatus,
+      currentMeetingStatus,
+      currentName,
+    } = this.state;
+
+    const ApplicantStatus = ({
+      provisional: { contribution, meetings },
+      name,
+      id,
+    }) => (
+      <tr>
+        <td>{name}</td>
+        <td>{contribution ? "Complete" : "Not Complete"}</td>
+        <td>{meetings ? "Complete" : "Not Complete"}</td>
+        {!settings.deliberationsOpen && (
```

Comment:
Just double checking, is this conditional correct or is this inversed for testing?